README.md

```diff
@@ -49,6 +49,24 @@ Download the latest release [https://github.com/andredumas/techan.js/releases/la
 npm install --save techan
 ```
 
+#### Webpack (ES6)
+```javascript
+// webpack.config.js
+config = {
+    ...,
+    resolve: {
+        ...,
+        alias: {
+            'techan': path.resolve('node_modules/techan/src/techan.js')
```

Comment:
This is not strictly necessary, but it allows webpack to work with the un-optimized source code instead of the ``dist/techan.js`` specified in ``package.json``'s main key. This way, end user's webpack configs can do whatever minification / code splitting / etc the deem necessary, instead of relying on the assumptions ingrained in the dist file. It also allows for getting accurate source maps should somebody prefer to do development on techan from their webpack-based project instead of the recommended approach.